linera-core/src/client/mod.rs

```diff
@@ -597,15 +597,6 @@ impl From<ViewError> for ChainClientError {
     }
 }
 
-impl From<LocalNodeError> for ChainClientError {
-    fn from(error: LocalNodeError) -> Self {
-        match error {
-            LocalNodeError::BlobsNotFound(blob_ids) => Self::BlobsNotFound(blob_ids),
```

Comment:
What would now return the `ChainClientError::BlobsNotFound` variant? I thought that a client deals w/ blobs as a result of interacting with a local or a remote node only.

Reply:
Yeah, our errors are not in an ideal state at the moment. `BlobsNotFound` currently gets propagated up. We only don't convert when it'll hide that it came from a validator or from the local node, which is what I think makes this propagation ok. This PR is fixing a case that was missed for the local node.
We do need to do a pass on how we're erroring because there's for sure places where for example we should be returning a `NodeError` to indicate the error actually comes from a validator and things like that, and we currently don't. I think @afck did some work on that front already

Reply:
True, good point, since we stopped doing these conversions and we're not propagating these errors past these boundaries we can probably clean a lot of these up I think 🤔 I can do that in a separate PR

Reply:
Can you create an issue for that and link in TODO comment?

Reply:
Was a quick PR so I just did it 😅 https://github.com/linera-io/linera-protocol/pull/2910